easybuild/easyconfigs/h/HeFFTe/HeFFTe-2.4.1-foss-2023b-CUDA-12.4.0.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'HeFFTe'
+version = '2.4.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://icl.utk.edu/fft'
+description = "Highly Efficient FFT for Exascale (HeFFTe) library"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+source_urls = ['https://github.com/icl-utk-edu/heffte/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['de2cf26df5d61baac7841525db3f393cb007f79612ac7534fd4757f154ba3e6c']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+
+dependencies = [
+    ('CUDA', '12.4.0', '', SYSTEM),
+    ('UCX-CUDA', '1.15.0', versionsuffix),
+]
+
+build_shared_libs = True
+
+configopts = "-DHeffte_ENABLE_FFTW=ON -DFFTW_ROOT=$EBROOTFFTW -DHeffte_ENABLE_MKL=OFF "
+configopts += "-DHeffte_ENABLE_CUDA=ON -DCUDA_TOOLKIT_ROOT_DIR=$EBROOTCUDA -DHeffte_ENABLE_GPU_AWARE_MPI=ON"
```

```suggestion
configopts += "-DHeffte_ENABLE_CUDA=ON -DCUDAToolkit_ROOT=$EBROOTCUDA -DHeffte_ENABLE_GPU_AWARE_MPI=ON"
```

`CUDA_TOOLKIT_ROOT_DIR` is old CMake.